src/planner/binder/statement/bind_update.cpp

```diff
@@ -202,10 +212,59 @@ BoundStatement Binder::Bind(UpdateStatement &stmt) {
 	// set the projection as child of the update node and finalize the result
 	update->AddChild(move(proj));
 
-	result.names = {"Count"};
-	result.types = {LogicalType::BIGINT};
-	result.plan = move(update);
-	this->allow_stream_result = false;
+	if (!stmt.returning_list.empty()) {
+		auto types = vector<LogicalType>();
```

Comment:
We would usually define this as `vector<LogicalType> types;`, no need to explicitly call the constructor here